openshift/release/create-release-branch.sh

```diff
@@ -19,6 +19,7 @@ git checkout -b "$target" "$release"
 git fetch openshift master
 git checkout openshift/master -- openshift OWNERS_ALIASES OWNERS Makefile content_sets.yml container.yaml
 make generate-dockerfiles
+make generate-p12n-dockerfiles
```

Comment:
This should be added to `update-to-head.sh` for good measure I think.

Reply:
As discussed offline, we'll only generate productization Dockerfiles on the release branch. They're not required anywhere else. As a result, there's no need to add to `update-to-head.sh`.